app/views/curation_concerns/base/_form_child_work_relationships.html.erb

```diff
@@ -0,0 +1,75 @@
+<%# Form UI behavior code and details;
+Code:
+  app/assets/javascripts/sufia/relationships_table.es6
+CSS:
+  table.relationships-api-enabled : Used by sufia JS app as selector to wire up the UI functionality
+  tr.new-row : The basic template row for cloning when user clicks "Add"
+  .btn-remove-row : Button to remove its parent TR from the table
+  .btn-add-row : Button to clone its parent TR and inject a new row into the table
+  input.new-form-control : Input field for client client side validation and additional features
+    .message.has-warning : Used to display UI errors related to input values and server errors
+HTML Properties:
+  table:
+    data-api-add-url : API url to call for adding a new row to this table
+    data-api-remove-url : API url to call for removing a row from this table
+    data-works-context : (parent|child)
+      this relates to the JSON returned from the server when successfully adding a new row.
+      See example API return:
+      {
+        "parent": {
+          "title":["dfdf"],
+          "path":"/concern/generic_works/9306sz37r",
+          "id":"9306sz37r"
+        },
+        "child": {
+          "title":["asdf"],
+          "path":"/concern/generic_works/2b88qc32v",
+          "id":"2b88qc32v"
+        }
+      }
+%>
+<div class="form-group multi_value optional managed">
+  <%= link_to "Attach New Work", polymorphic_path([main_app, :new, :curation_concerns, :parent, curation_concern.model_name.singular], parent_id: params[:id]), target: "_blank", class: 'btn btn-primary' %>
+
+  <table class="table table-striped related-files relationships-api-enabled"
+         data-api-add-url=<%="/api/generic_works/:id/add_parent/#{params[:id]}"%>
+         data-api-remove-url=<%="/api/generic_works/:id/remove_parent/#{params[:id]}"%>
```

Comment:
I think you're going to have to update these routes when you change the controller name.